src/SDKs/CognitiveServices/dataPlane/Vision/Vision/Microsoft.Azure.CognitiveServices.Vision.csproj

```diff
@@ -3,20 +3,15 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Vision APIs.</Description>
-    <VersionPrefix>1.0.1-preview</VersionPrefix>
-    <DefineConstants>$(DefineConstants);CODESIGN</DefineConstants>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <VersionPrefix>1.0.2-preview</VersionPrefix>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Vision</AssemblyName>
-    <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;Face API;REST HTTP client;Face SDK;Computer Vision;Vision;netcore451511</PackageTags>
+    <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;REST HTTP client;Face;Face API;Face SDK;Computer Vision;Computer Vision API;Computer Vision SDK;Vision;netcore451511</PackageTags>
     <PackageReleaseNotes>This is a preview release of the Cognitive Services Vision SDK. Included with this release is support for Face API and Computer Vision API.</PackageReleaseNotes>
   </PropertyGroup>
 
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Folder Include="Generated\Face\Models\" />
-    <Folder Include="Properties\" />
-  </ItemGroup>
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
```

Comment:
@lebronJ add the following comment above/below line 2 and line 16

Reply:
Done